supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -181,11 +195,21 @@ trait SharedFileSystemAsyncJobExecutionActor
   override def executeOrRecover(mode: ExecutionMode)(implicit ec: ExecutionContext) = {
     // Run now in receive, not in yet another Runnable.
     Future.fromTry(Try {
-      tellMetadata(startMetadataEvents)
-      executeScript()
+      mode match {
+        case Execute =>
+          tellMetadata(startMetadataEvents)
+          executeScript()
+        case Recover(recoveryId) =>
+          recoveryId match {
+            case job: SharedFileSystemJob => recoverScript(job)
+            case other => throw new RuntimeException(s"Unable to recover $other")
+          }
+        case _: UseCachedCall =>
+          ??? // TODO: PBE: Implement!
```

Comment:
Boo! New PBE TODO!

Actually, this will hopefully be a totally different `BackendJobCacheReadActor` (naming TBD) whose sole purpose is to clone a JobResult... So it should never reach the execution actor